docs/stable/serve/storage_aware_scheduling.md

```diff
@@ -0,0 +1,192 @@
+---
+sidebar_position: 0
+---
+
+# Storage Aware Scheduling
+
+## Pre-requisites
+To enable storage aware model loading scheduling, a hardward configuration file is required.
+For example, the following is a sample configuration file for two serverls:
+```bash
+echo '{
+  "0": {
+    "host_size": "32GB",
+    "host_bandwidth": "24GB/s",
+    "disk_size": "128GB",
+    "disk_bandwidth": "5GB/s",
+    "network_bandwidth": "10Gbps"
+  },
+  "1": {
+    "host_size": "32GB",
+    "host_bandwidth": "24GB/s",
+    "disk_size": "128GB",
+    "disk_bandwidth": "5GB/s",
+    "network_bandwidth": "10Gbps"
+  }
+}' > hardware_config.json
+```
+
+We will use Docker to run a ServerlessLLM cluster in this example. Therefore, please make sure you have read the [Docker Quickstart Guide](./getting_started/docker_quickstart.md) before proceeding.
```

Comment:
This relative path is not correct, should be `../getting_started/docker_quickstart.md`?

Reply:
Thanks! Pls check the updated version.